.github/CONTRIBUTING.md

```diff
@@ -62,28 +64,67 @@ Concourse will be running and reachable at
 To build and install the `fly` CLI from source, run:
 
 ```sh
-$ go install ./fly
+go install ./fly
 ```
 
 This will install a `fly` executable to your `$GOPATH/bin`, so make sure that's
 on your `$PATH`!
 
-Once `fly` is built, you can target the locally-running Concourse instance like
-so:
+Once `fly` is built, you can get a test pipeline running like this:
 
+**Log into the locally-running Concourse instance targeted as `dev`:**
 ```sh
-$ fly -t dev login -c http://localhost:8080 -u test -p test
+fly -t dev login -c http://localhost:8080 -u test -p test
 ```
 
-This will save the target as `dev`, but you can name it whatever you like.
+**Create an example pipeline that runs a hello world job every minute:**
+```sh
+fly -t dev set-pipeline -p example -c <(cat <<EOF
```

Comment:
We could check this in as an example .yml somewhere instead of inlining it. Maybe the beginnings of an "examples" folder?

Reply:
Can do!

Reply:
Done!